locales/lt-LT/messages.po

```diff
@@ -38,21 +39,29 @@ msgid "-- Any kind of file --"
 msgstr "-- Betkoks failas --"
 
 msgid "1 item imported from the given QTI/APIP XML Item Document."
-msgstr "1 elementas buvo importuotas iš pateikto QTI/APIP XML Elemento Dokumentas."
+msgstr ""
+"1 elementas buvo importuotas iš pateikto QTI/APIP XML Elemento Dokumentas."
 
 msgid "A choice must be selected"
 msgstr "Variantas privalo būti pasirinktas"
 
 msgid "A human interpretation of the variable's value."
 msgstr "Žmogiška kintamojo reikšmės interpretacija."
 
-msgid "A QTI component is not supported. The system returned the following error: %s\n"
-""
+msgid ""
+"A QTI component is not supported. The system returned the following error: %"
+"s\n"
```

Comment:
Why there are 3 lines? and is it ok if we divide variable type into 2 different strings?

Reply:
fixed